src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs

```diff
@@ -77,26 +87,23 @@ module internal ProjectFile =
 
         [| for line in lines do
             let fields = line.Split(',')
-            if fields.Length < 7 then raise (new System.InvalidOperationException(sprintf "Internal error - Invalid resolutions file format '%s'" line))
+            if fields.Length < 8 then raise (new System.InvalidOperationException(sprintf "Internal error - Invalid resolutions file format '%s'" line))
             else {
                 NugetPackageId = fields.[0]
                 NugetPackageVersion = fields.[1]
                 PackageRoot = fields.[2]
                 FullPath = fields.[3]
-                IsNotImplementationReference = fields.[4]
-                InitializeSourcePath = fields.[5]
-                NativePath = fields.[6]
+                AssetType = fields.[4]
+                IsNotImplementationReference = fields.[5]
+                InitializeSourcePath = fields.[6]
+                NativePath = fields.[7]
             }
```

Comment:
Can this be adjusted so it doesn't look like Java/JS code? e.g.

```fsharp
else
    {
        // ... the fields
    }
```

Reply:
Lol ...